official/utils/misc/distribution_utils.py

```diff
@@ -114,13 +114,13 @@ def get_distribution_strategy(distribution_strategy="default",
 
   distribution_strategy = distribution_strategy.lower()
   if distribution_strategy == "off":
-    if num_gpus > 1 or num_workers > 1:
+    if num_gpus > 1:
```

Comment:
So what would happen if `num_workers > 1` and user specifies `off`?

Reply:
They can use the parameter server training log built in Estimator to train their model.